src/components/FeatureFlagRenderer.js

```diff
@@ -55,11 +50,10 @@ export default class FeatureFlagRenderer extends Component {
   }
 
   _checkFeatureFlag () {
-    const { launchDarklyConfig: { clientId, user }, flagKey } = this.props;
-    const ldClient = ldBrowserInit(clientId, user);
+    const { ldClientWrapper, flagKey } = this.props;
```

Comment:
@jacobmoretti how would you feel about this: rather than getting `ldClientWrapper` directly from a prop, we import from `lib/utils` as a default value, but still allow it to be overridden by a prop (mostly for tests)?  That way we could avoid having to import and call it for every usage of `FeatureFlagRenderer`.